adam/language_specific/english/english_language_generator.py

```diff
@@ -560,7 +555,9 @@ def _translate_verb_argument(
                             preposition, filler_noun, role=CASE_SPATIAL
                         )
                 return (syntactic_role, filler_noun)
-            elif isinstance(filler, Region):
+            # As our filler is guaranteed to either be a Region or an Object at this point
```

Comment:
The value of being more defensive is that if this assumption ever changes in the future, we will get an exception alerting us this code needs to be updated.